src/eventMapping.ts

```diff
@@ -104,43 +104,45 @@ export function getCloudEvent(req: express.Request): CloudEventsContext | null {
     return cloudevent;
   }
 
-  console.log('Converting from background event to CloudEvent');
   const event = getBackgroundEvent(req);
-  if (event === null) {
-    console.error('Unable to extract background event');
-    return null;
+  if (!event) {
+    throw new Error('Unable to extract background event');
```

Comment:
More specific `Error` classes would be nice. I think `TypeError` makes sense here.